C/primitive/elements/primitive.c

```diff
@@ -37,14 +38,27 @@ enum TypeNamesForJets {
   sWord256,
   sSWord256,
   sWord32,
+  sWord512,
   word2TimesWord256,
   twoPlusWord4,
   word2TimesWord256PlusTwoPlusWord4,
   sWord2TimesWord256PlusTwoPlusWord4,
   sSWord2TimesWord256PlusTwoPlusWord4,
   twoTimesWord32,
   twoTimesWord64,
-  word256TimesWord512,
+  twoTimesWord256, point = twoTimesWord256,
```

Comment:
In 025d095a9f2853a37bc04d387b7a9400bd496b9c

lol, TIL that you can use `x = y` syntax in an enum definition where `y` refers to another enum variant